py-polars/polars/functions/lazy.py

```diff
@@ -794,9 +794,10 @@ def corr(
     b
         Column name or Expression.
     ddof
-        "Delta Degrees of Freedom": the divisor used in the calculation is N - ddof,
-        where N represents the number of elements.
-        By default ddof is 1.
+        Has no effect, do not use.
+
+        .. deprecated:: 1.17.0
+
```

Comment:
I'm not confident about choice the version written here. I just incremented from the current release version 1.16.0.